client/src/CommentBox.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { PropTypes } from 'react';
+import $ from 'jquery';
+import CommentForm from './AddCommentForm';
+import CommentList from './CommentList';
+
+class CommentBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: [] };
+  }
+
+  componentDidMount = () => {
+    this.loadComments();
+    setInterval(this.loadComments, this.props.pollInterval);
+  }
+
+  loadComments = () => {
+    $.ajax({
+      url: this.props.url.concat('/', this.props.poemId),
+      dataType: 'json',
+      cache: false,
+      success: (data) => {
+        this.setState({ data });
+      },
+      error: (xhr, status, err) => {
+        console.error(this.props.url, status, err.toString());
+      },
+    });
+  }
+
+  handleCommentSubmit = (comment) => {
+    const comments = this.state.data;
+    const mutatedComment = comment;
+    mutatedComment.poemId = this.props.poemId;
+    mutatedComment.id = Date.now();
+    const newComments = comments.concat([mutatedComment]);
```

Comment:
I got confused. The server doesn't _currently_ expect the poem ID as a path param. But I think it makes sense to add it because we're adding a comment to a particular poem.